tests/impl.cpp

```diff
@@ -2172,7 +2172,17 @@ result_t test_mm_rcp_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_rsqrt_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const float *_a = (const float *) impl.mTestFloatPointer1;
+
+    float f0 = 1 / sqrt(_a[0]);
+    float f1 = 1 / sqrt(_a[1]);
+    float f2 = 1 / sqrt(_a[2]);
+    float f3 = 1 / sqrt(_a[3]);
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 c = _mm_rsqrt_ps(a);
+
+    return validateFloatError(c, f0, f1, f2, f3, 0.01f);
```

Comment:
You shall explain the value `0.01f`.

Reply:
I was not convinced. Why `1%`?

Reply:
1% is the smallest error we can achieve under the current implementation.

Reply:
I defer to the review of @marktwtn .

Reply:
It is hard to decide the relative error when it comes to division and square root.
I do not have any good advice.

We might accept the current relative error and it needs to be considered carefully in the future pull request.

Reply:
Later, we might integrate the precise `rsqrt` implementation. At the moment, we have to address and explain the acceptable error rate.

Reply:
So should I open another PR which address the acceptable error here and leave this PR in current implementation

Reply:
Simply do it here.